web_m2x_options/static/src/js/form.js

```diff
@@ -482,7 +476,7 @@ odoo.define("web_m2x_options.web_m2x_options", function (require) {
             this._super(attrs);
             var limit = ir_options["web_m2x_options.field_limit_entries"];
             if (!_.isUndefined(limit)) {
-                attrs.limit = parseInt(limit);
+                attrs.limit = parseInt(limit, 10);
```

Comment:
I prefer the use of "unary plus" as it is more concise with the operation we want to do here.
```suggestion
                attrs.limit = +limit;
```

Reply:
Unary plus is not possible, because it is automatically replaced by pre-commit with:
Number(limit)